oxidation/libparsec/crates/core_fs/src/storage/chunk_storage.rs

```diff
@@ -69,10 +70,11 @@ pub(crate) trait ChunkStorageTrait {
         Ok(())
     }
 
-    fn close_connection(&self) {
-        self.conn().close();
+    async fn close_connection(&self) {
+        self.conn().close().await;
     }
 
+    #[allow(dead_code)]
     #[cfg(test)]
     async fn drop_db(&self) -> FSResult<()> {
         self.conn()
```

Comment:
If this function isn't used in test, remove it.

It's basic enough to recreate it if needed